src/Features/CSharp/Portable/FileHeaders/CSharpFileHeaderCodeFixProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.FileHeaders;
+
+namespace Microsoft.CodeAnalysis.CSharp.FileHeaders
+{
+    /// <summary>
+    /// Implements a code fix for file header diagnostics.
+    /// </summary>
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(CSharpFileHeaderCodeFixProvider))]
+    [Shared]
```

Comment:
it offends my sensibilities to have these attributes in this form that looks like they'll topple over :-0  can we either put `Shared` above the `Export`, or combine into one attribute?

Reply:
I can't make a change here:

* `Export` is the primary attribute, with `Shared` providing specific behavior within that context.
* I don't combine separate attributes into a single attribute list (same as I only declare one field on a line)